docs/Kube-API-examples.md

```diff
@@ -1,4 +1,48 @@
-#Assisted Installer Kube API CR examples
+# Hive Integration
+
+The goal of the Hive integration is to enable Assisted Installer capabilities on-premise in users' "Hub" clusters by installing clusters via Multi-cluster management, such as through [Hive](https://github.com/openshift/hive/) and [RHACM](https://github.com/open-cluster-management) (Red Hat Advanced Cluster Management).
+
+A full description of the enhancement is available [here](https://github.com/openshift/enhancements/blob/master/enhancements/installer/agent-based-installation-in-hive.md).
+
+For this integration, the Assisted Installer APIs are available via [CRDs](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)
+## CRD Types
+
+### [ClusterDeployment](https://github.com/openshift/hive/blob/master/apis/hive/v1/clusterdeployment_types.go)
+The ClusterDeployment CRD is defined in the Hive repository, where additional fields are added for Agent Based installation.
+In the ClusterDeployment, the user can specify requirements like Networking, number of Control Plane and Workers nodes and more.
+
+The installation will start automatically if the required number of hosts is available, the hosts are ready to be installed and the Agents are approved.
+
+### [InstallEnv](https://github.com/openshift/assisted-service/blob/master/internal/controller/api/v1alpha1/installenv_types.go)
+The InstallEnv CRD represents the configuration needed to create the discovery ISO.
+The user can specify proxy settings, ignition overrides and specify NMState labels.
+
+When the ISO is ready, an URL will be available in the CR.
+
+
+### [NMStateConfig](https://github.com/openshift/assisted-service/blob/master/internal/controller/api/v1alpha1/nmstate_config_types.go)
+The NMStateConfig contains network configuration that will applied on the hosts.
```

Comment:
NMState reference https://github.com/nmstate/kubernetes-nmstate/blob/master/README.md

Reply:
This repo is for kubernetes. I added referebce to https://github.com/nmstate/nmstate